sdk/keyvault/Azure.Security.KeyVault.Administration/src/Generated/AzureSecurityKeyVaultAdministrationModelFactory.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This doesn't add any value since all classes are constructable with public members. Can we suppress this for now? I wouldn't want to call the class name this anyway, as it doesn't follow our existing naming convention we used in other packages.

For example, can we use existing model renaming functionality to not only rename this but also hide it?

Reply:
I'll make it internal for now. Do you need an ability to be able to remove it completely?

Reply:
Don't need it, but definitely need the ability to rename it. That is not the naming convention we have used throughout all the languages SDKs for KV.

Reply:
Given what you told Mariana, could you rename this to just `AdministrationModelFactory`. That aligns with our other model factories in our other KV packages.